src/client/styles/modal/chat.css

```
@@ -57,6 +57,11 @@
   cursor: pointer;
 }
 
+.chat-send-button-disabled {
```

I think you can use the `:disabled` css selector:
```suggestion
.chat-send-button:disabled {
```